charts/clickhouse/values.yaml

```diff
@@ -287,7 +287,9 @@ coldStorage:
 # -- When the `installCustomStorageClass` is enabled with `cloud` set as `gcp` or `aws`,
 # it creates custom storage class with volume expansion permission.
 #
-installCustomStorageClass: false
+installCustomStorageClass: 
+  enabled: false
+  storageType: pd-ssd
```

Comment:
we cannot assume `pd-ssd` to be default storage type.

Reply:
Better keep it empty here and let the fallback storage type from template take care of it.

Otherwise, it would be breaking (or non-working) change for those upgrading.